src/style.css

```diff
@@ -1 +1,74 @@
 /* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+}
+
+li {
+  margin-left: 20px;
+}
+
+a {
+  cursor: pointer;
+  padding: 15px 0;
+  text-decoration: none;
+  color: black;
+  text-transform: uppercase;
+}
+
+a:hover {
+  text-decoration: underline;
+  text-decoration-color: #00acdc;
+  text-decoration-thickness: 2px;
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+img {
+  padding: 10px 0;
+}
+
+.is-active::after {
+  position: relative;
+  content: "";
+  display: block;
+  height: 4px;
+  background: #00acdc;
+  border-radius: 2px;
+  top: 21px;
```

Comment:
I don't think you should use exact pixels there
the line is at the bottom so use position absolute